Studies/HHBTag/CreateTrainingSkim.py

```diff
@@ -27,30 +33,38 @@ def createSkim(inFile, outFile, period, sample, X_mass, mpv, snapshotOptions):
     df = Baseline.RequestOnlyResolvedGenJets(df)
 
     df = Baseline.RecoLeptonsSelection(df)
-    df = Baseline.RecoJetAcceptance(df)
-    df = Baseline.RecoHttCandidateSelection(df)
+    # df = Baseline.RecoJetAcceptance(df)
+    df = Baseline.RecoHttCandidateSelection(df, config["GLOBAL"])
     df = Baseline.RecoJetSelection(df)
 
-    df = df.Define('genChannel', 'genHttCand.channel()')
-    df = df.Define('recoChannel', 'httCand.channel()')
+    df = df.Define('genChannel', 'genHttCandidate->channel()')
+    df = df.Define('recoChannel', 'HttCandidate.channel()')
 
     df = df.Filter("genChannel == recoChannel", "SameGenRecoChannels")
-    df = df.Filter("GenRecoMatching(genHttCand, httCand, 0.2)", "SameGenRecoHTT")
-    df = Baseline.RequestOnlyResolvedRecoJets(df)
+    # df = df.Filter("GenRecoMatching(genHttCandidate, HttCandidate, 0.2)", "SameGenRecoHTT")
```

Comment:
I think is better to require that genHttCandidate matches with the selected httCandidate

Reply:
But GenRecoMatching is a function that is no longer defined in the BaselineSelection.py (when I started looking at this, it was indeed defined). I'm not sure if I should require the matching in some way?

Reply:
It is still [there](https://github.com/cms-hh-bbtautau/Framework/blob/main/Common/BaselineRecoSelection.h#L124), but is commented for some reason: likely some compilation errors... Could you please fix it?